scripts/set-app-template-versions.js

```diff
@@ -106,17 +87,15 @@ const setVersion = (template, rootTmpDir) => {
       const json = JSON.stringify(packageJson, null, 2);
       fse.writeFileSync(packageJsonFile, json);
 
-      fse.writeFileSync(nvmrcFile, `v${newVersions.nodeVersion}`);
-
       travisYaml.node_js[0] = newVersions.nodeVersion;
-      fse.writeFileSync(travisYamlFile, yaml.stringify(travisYaml, null, 2));
+      return fse.writeFile(travisYamlFile, yaml.stringify(travisYaml, null, 2));
```

Comment:
it was a [tiny linting bug](https://cdn.zapier.com/storage/photos/e0cefa3ccf11f3664cccc46e35e3b283.png). We're expecting a result, so for one of the branches to return nothing (implicitly `undefined`) is bad form. Since we're already in a promise anyway, changing to the async version is super easy.